misc.go

```diff
@@ -27,6 +28,9 @@ type HTTPRequester interface {
 
 var customHTTPClient HTTPRequester
 
+// Default duration for rate limiting if Retry-After header does not exist/parse
+const defaultRetrySeconds int64 = 60
```

Comment:
@dvdplm added this as a constant up here, and changed it to 60.

Reply:
However, in my usage I mostly see 1 second delays, I wonder if instead of a default we should just raise a different error because we couldn't parse the retry-after header?

Reply:
You know what, I think you're right. There is really no "good" default here and if Slack sends a 429 but no retry-after value we can parse, then perhaps it's best to return an error and let applications decide what they want to do (retry after 1, 10 or a 100 secs, call the police or whatever fits their setup).

Reply:
I tend to agree, I personally have a strong policy of not lying or surprising the users in API design.

we could do two things here:
1) return the parse error directly and let the application developers deal with it.
2) return a different implementation of the retry error with a default value specified, and the underlying error that caused it to occur.

either way works, I'm partial to just return the error as its less code, and if it ever comes up in practice we can deal with it later.

Reply:
I went with option 2, just return the error.